_includes/meta.html

```diff
@@ -38,6 +38,18 @@
 <meta property="og:description" content="{{ description_rendered | xml_escape }}" />
 <meta property="twitter:description" content="{{ description_rendered | xml_escape }}" />
 
+<!-- Keywords -->
+{% if page.keywords %}
+  {% capture keywords_rendered %}
+    {{ page.keywords }}
+  {% endcapture %}
+{% else %}
+  {% capture keywords_rendered %}
+    {{ site.data.[page.lang].settings.global.keywords }}
```

Comment:
Do we have global keywords? If not, could we just render it if the page has keywords assigned?

```html
{% if page.keywords %}
  <meta name="keywords" content="{{ page.keywords | xml_escape }}" />
{% endif %}
```

Reply:
I have moved that tag into a conditional like you're suggesting.

Reply:
What I was envisioning is that we could get rid of the `capture keywords_rendered` code altogether, and that the snippet in my previous comment would be all the code we need in this template.

Reply:
I see. I was copying and changing the block for Description without fully understanding what was going on. It appears to be working now with just that block.

Reply:
Yeah, to be honest, I don't think that description logic is doing anything useful anyways, since the setting value it's trying to reference doesn't exist in the same way that `keywords` doesn't exist:

https://github.com/18F/identity-site/blob/d364b97e9f98e441c8da399563cc761154051551/_data/en/settings.yml#L191-L196